src/SourceBuild/tarball/content/repos/arcade.proj

```diff
@@ -2,7 +2,8 @@
   <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
 
   <PropertyGroup>
-    <BuildCommand>$(StandardSourceBuildCommand) $(StandardSourceBuildArgs)</BuildCommand>
+    <BuildCommandArgs>$(StandardSourceBuildArgs) $(FlagParameterPrefix)warnAsError $(ArcadeFalseBoolBuildArg)</BuildCommandArgs>
```

Comment:
Can you please update the projects that previously had no custom build args to utilize the new pattern - `<BuildCommandArgs>$(StandardSourceBuildArgs)</BuildCommandArgs>`